app/src/main/java/com/example/mytrainingpal/screens/Calendar.kt

```diff
@@ -8,24 +8,28 @@ import androidx.compose.foundation.shape.RoundedCornerShape
 import androidx.compose.material.MaterialTheme
```

Comment:
The lazy.items import is unnecessary, I think?

Reply:
:+1 I'll let the IDE remove all unnecessary imports

Reply:
Update: `MaterialTheme` is still being used in the file. E.g. line 49